pkg/modules/resiliency.go

```diff
@@ -129,16 +130,34 @@ func getResiliencyEnv(resiliencyModule csmv1.Module, driverType csmv1.DriverType
 	return ""
 }
 
-func getResiliencyArgs(m csmv1.Module, mode string) []string {
+func applyModuleToSideCar(component csmv1.ContainerTemplate, container *acorev1.ContainerApplyConfiguration) {
```

Comment:
Please add appropriate comments above the function

Reply:
done